addons/kibana/kibana.yaml

```diff
@@ -29,14 +29,13 @@ spec:
     - matchLabels:
         kubeaddons.mesosphere.io/name: elasticsearch
   chartReference:
-    chart: stable/kibana
-    version: 3.2.7
+    chart: elastic/kibana
```

Comment:
```suggestion
    chart: kibana
```

Reply:
the reason for this is because of the way kubeaddons defines the name of a repo (repoName) and how the chart's name is used. Otherwise CI will fail.